serverless/pages/spaces.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: /serverless/spaces
+title: Spaces
+description: Organize your project and objects into multiple spaces
+---
+
+Spaces enable you to organize your dashboards and other saved
+objects into meaningful categories. Once inside a space, you see only
+the dashboards and saved objects that belong to that space.
+
+When you create and enter a new project, you're using the default space of that project.
+
+You can identify the space you're in or switch to a different space from the header.
+
+<DocImage size="l" url="../images/space-breadcrumb.png" alt="Space breadcrumb" />
+
+You can view and manage the spaces of a project from the **Spaces** page in **Management**.
+
+## Required permissions
+
+You must have the admin role on the project to manage its **Spaces**.
+
+## Create or edit a space
+
+You can create as many spaces as you need. 
+
+1. Click *Create space* or select the space you want to edit.
+
+2. Provide:
+    - A meaningful name and description for the space.
+    - A URL identifier. The URL identifier is a short text string that becomes part of the ((kib)) URL. ((kib)) suggests a URL identifier based on the name of your space, but you can customize the identifier to your liking. You cannot change the space identifier later.
+
+3. Customize the avatar of the space to your liking.
+
+4. Save the space.
+
+((kib)) also has an [API](https://www.elastic.co/docs/api/doc/serverless)
+if you prefer to create spaces programmatically.
+
+<DocIf condition={"((serverlessCustomRoles))" === "true"}>
+## Customize access to space
+
+As an administrator, you can define custom roles with specific access to certain spaces and features in a project. Refer to <DocLink slug="/serverless/custom-roles"/>.
+</DocIf>
+
+## Delete a space
+
+Deleting a space permanently removes the space and all of its contents.
+Find the space on the *Spaces* page and click the trash icon in the Actions column.
+
+You can't delete the default space, but you can customize it to your liking.
+
+## Move saved objects between spaces
+
+To move saved objects between spaces, you can <DocLink slug="/serverless/saved-objects">export and import objects</DocLink>.
```

Comment:
It looks like the copy action is also available in serverless.

<img width="506" alt="Screenshot 2024-09-30 at 4 49 43 PM" src="https://github.com/user-attachments/assets/037e7fcc-a629-408e-a7c2-46e82379e391">

<img width="621" alt="Screenshot 2024-09-30 at 4 49 36 PM" src="https://github.com/user-attachments/assets/2590462c-0c8c-4da3-86a2-e75053a0b3a2">

Reply:
Oh good, I couldn't find it when I did my testing few weeks ago. I'll re-add it

Reply:
Added in latest commit, also added  the missing "copy to spaces" section in the saved objects page that probably got removed while spaces were unavailable.